components/nav/NavUser.vue

```diff
@@ -15,7 +19,21 @@
       <UserSwitcher ref="switcher" @click="hide()" />
     </template>
   </VDropdown>
-  <button v-else btn-solid text-sm px-2 py-1 text-center xl:hidden @click="openSigninDialog()">
-    {{ $t('action.sign_in') }}
-  </button>
+  <template v-else>
+    <button
+      v-if="singleInstanceServer"
+      flex="~ row" gap-x-1 items-center justify-center btn-solid text-center text-sm px-2 py-1 xl:hidden
+      :disabled="busy"
+      @click="oauth()"
+    >
+      <span v-if="busy" aria-hidden="true" block animate animate-spin preserve-3d class="rtl-flip">
+        <span block i-ri:loader-2-fill aria-hidden="true" />
+      </span>
+      <span v-else aria-hidden="true" block i-ri:login-circle-line class="rtl-flip" />
+      {{ $t('action.sign_in') }}
```

Comment:
I think we should make this clearer for people who have used Elk before but might not realise:

In English, something like: 'Sign in to <strong>m.webtoo.ls</strong>'